### ISSUE TITLE
Vault k8s auth is not working proerly

### ISSUE THREAD

**Issue by evilezh** [banzaicloud/koperator#289]
Code is written assuming default mount path for auth  - api call like /v1/auth/kubernetes/login
Meanwhile it is true only if I have
1. only one k8s cluster
2. and I use default mount points
3. or I use separate vault installation per cluster

Normally mount points to match cluster ids would be used , which automatically leads to different mount points. Api calls would be something like: e.g. /v1/auth/eks-dev/login, /v1/auth/kubernetes-uat/login etc.

**Comment by stoader** [comment]
@evilezh thanks for reporting this issue. Are you open to take take a stab at it and submit a PR that addresses this issue?

**Comment by evilezh** [comment]
Hi, sorry but not this time. I've moved on, as needed kafka cluster right now and was looking into options apart of static helm chart, which I had already.
It was more about giving fair feedback on product. To explain what prevented me to use it. I've also created another ticket a bit more information.
I was also looking into cert-manager option, but couldn't figure out how that shall work or not. And from deps there was linked older version of it rather than latest - so I assumed, it wouldn't work for me either.
